modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -64,6 +65,7 @@
         :status="searchField.status" :value="searchField.value"
         :modifiers="['small']"
         ref="search"
+        class="apos-manager-toolbar__search"
```

Comment:
bit confused by this change to be honest
Are we using this new class?

Reply:
seems not, will remove